layouts/partials/display-events.html

```diff
@@ -12,12 +12,20 @@
   {{ $chunks = merge $chunks (dict (string $current_key) $chunk) }}
 {{ end }}
 
-{{ range $chunkIndex, $chunk := $chunks }}
-  <div class="carousel-item {{ if eq $chunkIndex "0" }}active{{ end }}">
+{{ $scratch := .scratch }}
```

Comment:
Sorry I missed this PR!

Are these scratch variables still necessary given the script? IIUC they're not read anywhere and wouldn't be useful for changing the appearance of the HTML post-Hugo-build.